Cargo.toml

```diff
@@ -8,6 +8,6 @@ edition = "2018"
 bitflags = "1"
 
 [build-dependencies]
-bindgen = "0.51.0"
+bindgen = "*"
```

Comment:
I don't think we should be unpinning. I think we should bump the pin and maybe add depndabot.